### PR TITLE
Create JacketsMenPage

### PR DESCRIPTION
**Pull request by EstebanGlober**
Create the page of Jacket men
-Create the Attributes
-Create Methods

### CODE REVIEW

File: src/main/java/pages/JacketsMenPage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    public JacketsMenPage (WebDriver driver,Actions action) {
+        this.driver = driver;
+        this.action = action;
+    }
+
+    public void hoverCardOfJacket(){
+        WebElement cardOfJackets = driver.findElement(By.className("product-item-info"));
+    }
+    public void  clickSizeButtonS(){
```
Comment (larry-padilla) [inline src/main/java/pages/JacketsMenPage.java:28]: the last letter is not necessary in the method name

File: src/main/java/pages/JacketsMenPage.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    public void clickIconCart(){
+        driver.findElement(iconCart).click();
+    }
+
+    public  void clickCheckoutButton(){
+        driver.findElement(checkoutButton).click();
+    }
+
```
Comment (larry-padilla) [inline src/main/java/pages/JacketsMenPage.java:51]: this space is not necessary

**Review by larry-padilla — COMMENTED** [review]
you can resolve these comments in the following PR

File: src/main/java/pages/JacketsMenPage.java (hunk 1 above)
Comment (EstebanGlober) [reply src/main/java/pages/JacketsMenPage.java:28]: Done

File: src/main/java/pages/JacketsMenPage.java (hunk 2 above)
Comment (EstebanGlober) [reply src/main/java/pages/JacketsMenPage.java]: Done

**Review by larry-padilla — APPROVED** [review]

**Pull request merged**